### PR TITLE
Fix missing dependency version locking

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

File: openshift-ansible.spec [hunk 1]
```diff
@@ -224,7 +224,7 @@ BuildArch:     noarch
 # ----------------------------------------------------------------------------------
 %package lookup-plugins
 Summary:       Openshift and Atomic Enterprise Ansible lookup plugins
-Requires:      %{name}
+Requires:      %{name} = %{version}
```
Comment (detiber) [inline openshift-ansible.spec]: looks like filter-plugins, roles are missing this as well.

Comment (sdodson) [inline openshift-ansible.spec]: Thanks, got em all now.

**Comment by detiber** [comment]
:+1:

**Comment by sdodson** [comment]
@brenton this is ready

**Pull request merged**